lib/home_page_ui.dart/my_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+class MyButton extends StatelessWidget {
```

Comment:
this should be meaningful, 

maybe call it ImageButton?

Reply:
done